### ISSUE TITLE
`pck` [?] command

### ISSUE THREAD

**Issue by tryoxiss** [Kubulambula/Godot-GDShell#25]
I don't really know what this command should be called, but it would be nice to have a simple command that loads a PCK at runtime given a file path. It would be done with [`ProjectSettings.load_resource_pack(file)`](https://docs.godotengine.org/en/stable/classes/class_projectsettings.html#class-projectsettings-method-load-resource-pack), so it would be simple to implement.

~~Possibly `res://` could be provided, so you just need to specify the file path within that.~~ **It is actually reccommended you load mods from `user://`, so if we provide anything that should be it.**

Sorry for posting two issues in a row, just some things I was thinking of.

**Comment by Kubulambula** [comment]
I can see this being used for some niche problems with game patching or DLCs, but it would be unsafe to have it deployed in release builds. We could make an export script that would handle this.

I opened a [proposal](https://github.com/Kubulambula/Godot-GDShell/issues/26) with the first idea that came to my mind.


> Sorry for posting two issues in a row, just some things I was thinking of.

No problem at all! Whilst don't have time to work on this project, I'm really happy, that someone is paying attention to it!

**Comment by tryoxiss** [comment]
Yeah, I release its very nieche but seems easy to implement (I believe I got a working version of it already actually, but its hard to test) and helps with the ocasional debugging thing. I thought of it because I was working on a basic mod loader for a project template and realised it would be nice to load them arbitrary.

**Comment by Kubulambula** [comment]
It's very simple as it essentially just calls one engine method with the correct arguments. This should be really easy and quick to implement.
